tests/tests/fee_tests.cpp

```diff
@@ -213,6 +213,129 @@ BOOST_AUTO_TEST_CASE(asset_claim_fees_test)
    FC_LOG_AND_RETHROW()
 }
 
+BOOST_AUTO_TEST_CASE(asset_claim_pool_test)
+{
+    try
+    {
+        ACTORS((alice)(bob));
+        // Alice and Bob create some user issued assets
+        // Alice deposits BTS to the fee pool
+        // Alice claimes fee pool of her asset and can't claim pool of Bob's asset
+
+        const share_type core_prec = asset::scaled_precision( asset_id_type()(db).precision );
+        const auto& fees = *db.get_global_properties().parameters.current_fees;
+
+        // return number of core shares (times precision)
+        auto _core = [&core_prec]( int64_t x ) -> asset
+        {  return asset( x*core_prec );    };
+
+        const asset_object& alicecoin = create_user_issued_asset( "ALICECOIN", alice,  0 );
+        const asset_object& aliceusd = create_user_issued_asset( "ALICEUSD", alice, 0 );
+
+        asset_id_type alicecoin_id = alicecoin.id;
+        asset_id_type aliceusd_id = aliceusd.id;
+        asset_id_type bobcoin_id = create_user_issued_asset( "BOBCOIN", bob, 0).id;
+
+        // prepare users' balance
+        issue_uia( alice, aliceusd.amount( 20000000 ) );
+        issue_uia( alice, alicecoin.amount( 10000000 ) );
+
+        transfer( committee_account, alice_id, _core(1000) );
+        transfer( committee_account, bob_id, _core(1000) );
+
+        enable_fees();
+
+        auto claim_pool = [&]( const account_id_type& issuer, const asset_id_type& asset_to_claim,
+                              const asset& amount_to_fund, const asset_object& fee_asset  )
```

Comment:
Only big objects need to use references (&). Small objects like `asset` or `asset_id_type` don't need it.

Reply:
Makes sense for asset (because that's an object), but not for ..._id_type (as that's basically an int64).